server/application/application.proto

```diff
@@ -151,6 +162,15 @@ service ApplicationService {
 		option (google.api.http).get = "/api/v1/applications/{name}/manifests";
 	}
 
+	// DiffManifests compares two manifests and returns the diff
+	rpc DiffManifests(ApplicationManifestDiffRequest) returns (ApplicationManifestDiffResponse) {
+		option (google.api.http) = {
+			post: "/api/v1/applications/manifest"
```

Comment:
I think we should change the API to be:

GET `/api/v1/applications/my-app/diff?resourceName=my-pod&apiVersion=v1&kind=Pod`

Which presumes a diff between config and live, without having to supply the whole manifests .

In the future, this can be improved to support diffing a resource across apps:

GET `/api/v1/applications/my-app/diff?resourceName=my-pod&apiVersion=v1&kind=Pod&compareto={appname2}`